### PR TITLE
Pull request for code review - Mobile menu + JS

### PR DESCRIPTION
**Pull request by wickathou**
This design takes the structure from template 1 on [this Figma file](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) with a few changes on the icons, background and font sizing.

For this PR, a JS script is used to control the actions of the menu button on the mobile view

This PR, features contributions from @Eugeneiyukhun

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,31 +5,32 @@
...
+      <span class="mv" id="mobile-menu">
+        <img class="menu btn-menu" src="assets/images/menu-template.svg" alt="Navigation menu">
       </span>
       <span class="dv">
         <ul class="g20px xul">
-          <li class="link-text"><a href="http://">Portfolio</a></li>
-          <li class="link-text"><a href="http://">About</a></li>
-          <li class="link-text"><a href="http://">Contact</a></li>
+          <li class="link-text"><a href="#portfolio">Portfolio</a></li>
+          <li class="link-text"><a href="#about">About</a></li>
```
Comment (thecodechaser) [inline index.html:27]: - [ ] When users click on the About option, The about section of the page should be displayed but it's displaying the hero section. Please make sure that when users click on the About option the below part of the page should be displayed.

image

File: style.css [hunk 2]
```diff
@@ -512,3 +520,25 @@ form textarea {
...
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  z-index: 100;
+  background-color: #6070ff8e;
+}
```
Comment (thecodechaser) [inline style.css:536]: - [ ] Please make sure the background color matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=4DHMTBKEmxhBQW4R-0)  design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @wickathou, and @Eugeneiyukhun👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @wickathou 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] It would be better for the background not to be scrollable when the mobile nav is open, this will create a better user experience  👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**